shared/chat/conversation/attachment-fullscreen/index.desktop.tsx

```diff
@@ -69,9 +69,9 @@ class _Fullscreen extends React.Component<Props & Kb.OverlayParentProps, State>
         <Kb.Box style={styles.container}>
           <Kb.HotKey hotKeys={this.hotKeys} onHotKey={this.onHotKey} />
           <Kb.Box style={styles.headerFooter}>
-            <Kb.Text lineClamp={2} type="BodySemibold" style={{color: Styles.globalColors.black, flex: 1}}>
+            <Kb.Markdown lineClamp={2} style={{flex: 1}} meta={{message: this.props.message}}>
```

Comment:
Not a big deal but there's also `Styles.globalStyles.flexOne`

Reply:
oh neat! i can change that, seems better for stylistic reasons